charts/app-of-apps/values-integration.yaml

```diff
@@ -57,3 +57,40 @@ ckanHelmValues:
     count: 1
   externalSecret:
     enabled: true
+
+datagovukHelmValues:
+  environment: integration
+  publish:
+    count: 1
+    args: ["bundle exec rake db:migrate db:seed && bundle exec sidekiq"]
+    probes:
+      enabled: false
+    config:
+      ckanReleaseName: "ckan"
+      sqlalchemyUrlSecretKeyRef:
+        name: ckan
+        key: publish_sqlalchemy_url
+      dbHost: ""
+      redis:
+        host: "shared-redis-govuk.eks.integration.govuk-internal.digital"
+        port: "6379"
+        db_number: "1"
+      org:
+        schedule: "*/10 * * * *"
```

Comment:
Optional: this seems quite frequent for a cronjob. If it's a lesser-of-evils thing for now then no probs — just mentioning in case there happens to be an easy alternative. No worries if not.

Reply:
this is copied across from the current settings, probably doesn't need to be so frequent for integration but I think users expect the data within 30 minutes or so from CKAN to Find